catalog/pytest.ini

```diff
@@ -18,6 +18,8 @@ addopts =
     --disable-socket
     --allow-unix-socket
 
+log_cli = True
```

Comment:
Is this left-over from debugging the tests?
Tests run fine locally for me, but with this, the output does not look correct (the progress bar is not displayed as expected)

Reply:
Left over, yes, I'll remove it.

```suggestion
```